gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_6_0/01_add_api_type_to_plans_table.yml

```diff
@@ -0,0 +1,11 @@
+databaseChangeLog:
+  - changeSet:
+      id: 4.6.0_add_api_type_to_plans_table
+      author: GraviteeSource Team
+      changes:
+        - addColumn:
+            tableName: ${gravitee_prefix}plans
+            columns:
+              - column:
+                  name: api_type
+                  type: VARCHAR(255)
```

Comment:
what about keep `nvarchar(64)` same as the one in API ?

Reply:
Done